packages/connect-common/scripts/check-firmware-revisions.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+PARENT_PATH=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+
+echo $PARENT_PATH
+
+BRANCH="main"
+DIST="."
+REPO_DIR_NAME="trezor-firmware-for-revision-check"
+
+cd ..
+
+if test -d ./$REPO_DIR_NAME; then
+    echo "$REPO_DIR_NAME directory exists"
+else
+    echo "$REPO_DIR_NAME directory does not exist"
+    git clone https://github.com/trezor/trezor-firmware.git $REPO_DIR_NAME
+fi
+
+cd $REPO_DIR_NAME
+git fetch origin
+git checkout "$BRANCH"
+git reset "origin/$BRANCH" --hard
+
+DATA=`cat $PARENT_PATH/../files/firmware/t1b1/releases.json | jq -r '.[] | .version |= join(".") | .firmware_revision + "%" + .version'`
```

Comment:
I am, not sure if `jq` would be available in github bash env. 

- [ ] Todo: check, solve with somebody (adminkos?)

Reply:
it works